crates/evm/evm/src/inspectors/stack.rs

```diff
@@ -551,19 +551,19 @@ impl InspectorStack {
         let (result, address, output) = match res.result {
             ExecutionResult::Success { reason, gas_used, gas_refunded, logs: _, output } => {
                 gas.set_refund(gas_refunded as i64);
-                gas.record_cost(gas_used);
+                let _record_gas = gas.record_cost(gas_used);
```

Comment:
can we remove the unused var?

Reply:
Added this due to clippy, can remove or make it `let _ = gas.record_cost()`.

Reply:
This is fine because it's a new Gas instance, I added must_use for use within EVM instructions.